lib/propolis/src/block/crucible.rs

```diff
@@ -278,6 +280,10 @@ impl CrucibleBackend {
             );
         }
     }
+
+    pub async fn volume_is_active(&self) -> io::Result<bool> {
```

Comment:
Why make the trip through std::io::Error (and Result) here, vs letting the CrucibleError through?

Reply:
good point, done in 473bdfe